src/main/java/com/thesamet/gerrit/plugins/saml/SamlConfig.java

```diff
@@ -74,6 +78,19 @@ public String getEmailAddressAttr() {
     return emailAddressAttr;
   }
 
+  public String getMaxAuthLifetimeAttr() { return maxAuthLifetimeAttr; }
+
+  public int getMaxAuthLifetime() {
+    int maxLifetime;
+    try {
+      maxLifetime = Integer.parseInt(getMaxAuthLifetimeAttr());
+    } catch (NumberFormatException nfe) {
+      SamlWebFilter.logError("Error reading \"maxAuthLifetime\" attribute in gerrit.config. Please use digits only");
```

Comment:
Please add a logger for this class instead of using the logger in SamlWebFilter.